src/amo/components/HomeHero/index.js

```diff
@@ -0,0 +1,93 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+
+import Link from 'amo/components/Link';
+import translate from 'core/i18n/translate';
+import Hero from 'ui/components/Hero';
+import type { I18nType } from 'core/types/i18n';
+import Card from 'ui/components/Card';
+import Icon from 'ui/components/Icon';
+
+import './styles.scss';
+
+type InternalProps = {|
```

Comment:
You could spread an empty `Props` type maybe, because `Props` is what you should pass to ` React.ComponentType` at the bottom of the page.